cmd/livepeer/livepeer.go

```diff
@@ -159,7 +160,13 @@ func main() {
 	orchWebhookURL := flag.String("orchWebhookUrl", "", "Orchestrator discovery callback URL")
 	detectionWebhookURL := flag.String("detectionWebhookUrl", "", "(Experimental) Detection results callback URL")
 
-	flag.Parse()
+	// Config file (used instead of flags)
+	_ = flag.String("config", "", "Config file in the format 'key value', flags take precedence over the config file")
+	ff.Parse(flag.CommandLine, os.Args[1:],
+		ff.WithConfigFileFlag("config"),
```

Comment:
I think we'll need something like this to enable environment variables?
```suggestion
		ff.WithConfigFileFlag("config"),
		ff.WithEnvVarPrefix("LP"),
```

Reply:
I didn't know we also want to use env variables. But if yes, then right.